filesystem/Folder.php

```diff
@@ -165,6 +165,7 @@ public function syncChildren() {
 
 				// Check allowed extensions, unless admin users are allowed to bypass these exclusions
 				if($checkExtensions
+					&& !is_dir("$baseDir/$actualChild")
```

Comment:
On line 199 it does `$baseDir . $actualChild` to concatenate these. Does does $baseDir have a trailing slash?